File: 96_Dodatno/01_C/Annoying.c
#include <stdio.h>

#define MAX_INPUT 100

int main(int argc, char *argv[]){
  // Infinate loop that repeats the users input back to them, if there is any (with many errors waiting to happen)
  char userInput[MAX_INPUT];

  while ((fgets(userInput, MAX_INPUT, stdin) != NULL) && (userInput[0] != '\n')){
    printf("%s", userInput);
  }

  return 0;
}

File: 96_Dodatno/00_Java/Annoying.java
import java.util.*;

public class Annoying {
  public static void main(String[] args){
    // Infinate loop that repeats the users input back to them, if there is any
    Scanner sc = new Scanner(System.in);

    String userInput = "";
    while(true){
      userInput = sc.nextLine();

      if (userInput.isEmpty()){
        break;
      }

      System.out.println(userInput);
    }

    sc.close();
  }  
}
